extensions/amp-viewer-integration/0.1/messaging/messaging.js

```diff
@@ -297,8 +383,8 @@ export class Messaging {
    */
   sendMessage_(message) {
     const /** Object<string, *> */ finalMessage = Object.assign(message, {});
-    if (this.token_) {
-      finalMessage['messagingToken'] = this.token_;
+    if (this.token_ && !this.verifyToken_) {
+      finalMessage.messagingToken = this.token_;
```

Comment:
We need to decide on quotes or direct reference here. And I'd prefer quotes since it's a plain-text message that goes over the boundary of iframes/channels/etc.

Reply:
I ended up switching to use `parseMessage` and direct reference everywhere, as this also gives us stronger typing.